docs/api.yml

```diff
@@ -0,0 +1,307 @@
+                        type:
+                          type: string
+                          enum: [range, point]
+                        content:
+                          type: string
+                          description: Short info about the task
+                        title:
+                          type: string
+                          format: html
+                          description: Detailed information about the task
+                        style:
+                          type: string
+                          description: Style attribute to apply on the rendered task
+                  groups:
+                    type: array
+                    items:
+                      type: object
+                      required:
+                        - id
+                        - content
+                        - order
+                      properties:
+                        id:
+                          type: string
+                        content:
+                          type: integer
+                        order:
+                          type: integer
+  /job/{job_id}/cancel:
+    post:
+      summary: Cancel the job with the specified job_id
+      parameters:
+        - name: job_id
+          in: path
+          schema:
+            type: string
+            format: uuid
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/empty:
+    post:
+      summary: Empty the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/delete:
+    post:
+      summary: Delete the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queues.json:
+    get:
+      summary: List Queues and their state
+      responses:
+        200:
+          description: |
+            Return object containing a list of queues and the `[failed]`,
+            `[finished]` and `[running]` registries, the number of elements
+            in them and an url for more details on them.
```

Comment:
I googled, found a not very reputable source but I agree
>Although the U at the first of URL is a vowel, the abbreviation is pronounced “you-are-ell.” Because the letter Y is a consonant in this case, use a rather than an.

Reply:
```suggestion
            in them and a URL for more details on them.
```